### ISSUE TITLE
18.03 Zero Hydra Failures - Darwin edition

### ISSUE THREAD

**Issue by vcunat** [NixOS/nixpkgs#36454]
See [the generic description](https://github.com/NixOS/nixpkgs/issues/36453).   This mainly makes a place to discuss Darwin-specific issues for ZHF.

**Comment by knedlsepp** [comment]
We still got a few evaluation errors left:
Here's the list ordered by how many packages are affected by the respective package not evaluating:
```
nix-shell --pure -p curl go-pup --run "curl -k -s https://hydra.nixos.org/jobset/nixpkgs/nixpkgs-18.03-darwin#tabs-errors | pup pre | grep 'not supported' | sort | uniq -c | sort -h
```

**Comment by LnL7** [comment]
I've gone through the entire list now a decent amount of packages that remain have always been broken and should probably be marked as linux only.

https://hydra.nixos.org/eval/1444265?compare=nixpkgs-17.09-darwin#tabs-still-fail

**Comment by vcunat** [comment]
Thanks for all the fixes in 18.03 round!  (I'm late, I know.)

The next ZHF iteration: https://github.com/NixOS/nixpkgs/issues/45961
